_includes/tutorials/multi-joins/ksql/markup/dev/print-topic.adoc

```diff
@@ -0,0 +1,13 @@
+Finally, let's see what's available on the underlying Kafka topic for the new stream. We can print that out easily.
+
++++++
+<pre class="snippet"><code class="sql">{% include_raw tutorials/multi-joins/ksql/code/tutorial-steps/dev/print-topic.sql %}</code></pre>
++++++
+
++++++
+<pre class="snippet"><code class="shell">{% include_raw tutorials/multi-joins/ksql/code/tutorial-steps/dev/expected-print-topic.log %}</code></pre>
++++++
+
+Notice that the key for each message is the Item ID of the order.  This is the result of the join with the items table being the last join for our `CREATE STREAM` command. The key of the last join will become the key of the records in the underlying topic.
+
+Exit the ksqlDB CLI with `Ctrl-d`.
```

Comment:
nit: Exit the ksqlDB CLI by typing `exit`

Reply:
Updated in latest